app/resources/api/v2/plate_purpose_resource.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API representation of Plate
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class PlatePurposeResource < BaseResource
```

Comment:
It is the model being wrapped by this resource. We are passing a custom attribute for now, instead of exposing the full asset_shape relationship. I don't see an issue with that.